compiler/rustc_resolve/src/lib.rs

```diff
@@ -108,7 +107,7 @@ enum Scope<'a> {
     DeriveHelpersCompat,
     MacroRules(MacroRulesScopeRef<'a>),
     CrateRoot,
-    Module(Module<'a>),
+    Module(Module<'a>, /* poisoned */ bool),
```

Comment:
Can you add a comment explaining that this is used for `PROC_MACRO_DERIVE_RESOLUTION_FALLBACK`?